pkg/webhooks/webhooks.go

```diff
@@ -208,3 +210,16 @@ func HealthProbe(ctx context.Context) healthz.Checker {
 		return nil
 	}
 }
+
+func ValidateConversionEnabled(ctx context.Context, kubeclient client.Client) {
+	// allow context to exist longer than cache sync timeout
+	listCtx, cancel := context.WithTimeout(ctx, 130*time.Second)
+	defer cancel()
+	// sleep for cache hydration
+	// wait for cache to sync, controller-runtime defaults to 120 seconds
+	time.Sleep(120 * time.Second)
```

Comment:
Why not continually poll and just exit out as soon as we succeed?

Reply:
+1 on polling. Also, if we do keep the sleep, should it be greater than the controller-runtime cache invalidation period to prevent races, rather than equal.